ansible_collections/arista/avd/requirements.txt

```diff
@@ -5,7 +5,7 @@ cvprac>=1.0.7
 paramiko>=2.7.1
 jsonschema>=4.5.1
 requests>=2.25.1
-PyYAML>=5.4.1
+PyYAML>=6.0.0
```

Comment:
Please do not change requirements in a regular feature PR.

Reply:
reverted back,  was using to test to get the libyaml to work